src/Microsoft.Identity.Client/Platforms/iOS/iOSPlatformProxy.cs

```diff
@@ -156,5 +157,15 @@ public override int GetMatsOsPlatformCode()
         }
 
         protected override IFeatureFlags CreateFeatureFlags() => new iOSFeatureFlags();
+
+        public override IBroker CreateBroker()
+        {
+            if (base.CreateBroker() == base.OverloadBrokerForTest)
```

Comment:
Why do you need to call base.CreateBroker() instead of just checking if OverloadBrokerForTest is non-null?  If someone has set the test hook, shouldn't you just use it?

Reply:
definitely . thanks.

---
In reply to: [308433527](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1295#discussion_r308433527) [](ancestors = 308433527)